src/common/constants.js

```diff
@@ -0,0 +1,77 @@
+const affiliate_signin_url = 'https://login.deriv.com/signin.php'
+const affiliate_signup_url = 'https://login.deriv.com/signup.php'
+const application_id = 'f0aef779-d9ec-4517-807e-a84c683c4265'
+const besquare_signup_url =
+    'https://deriv.zohorecruit.com/jobs/Careers/590522000011882533/BeSquare-Graduate-Trainee?source=CareerSite'
+const binary_url = 'https://binary.com'
+const blog_url = 'https://blog.deriv.com'
+const brand_name = 'Deriv'
+const cfd_warning_height = {
+    desktop: 8,
+    tablet: 12,
+}
+const client_token = 'pubc42fda54523c5fb23c564e3d8bceae88'
+const community_url = 'https://community.deriv.com'
+const deriv_app_languages = ['id', 'pt', 'es']
+const deriv_app_url = 'https://app.deriv.com'
+const deriv_blog_url = 'https://blog.deriv.com'
+const deriv_bot_app_url = 'https://app.deriv.com/bot'
+const deriv_cookie_domain = 'deriv.com'
+const deriv_dp2p_app_url = 'https://app.deriv.com/cashier/p2p'
+const deriv_status_page_url = 'https://deriv.statuspage.io'
+const dmt5_android_url =
+    'https://download.mql5.com/cdn/mobile/mt5/android?server=Deriv-Demo,Deriv-Server'
+const dmt5_ios_url = 'https://download.mql5.com/cdn/mobile/mt5/ios?server=Deriv-Demo,Deriv-Server'
+const dmt5_linux_url = 'https://www.metatrader5.com/en/terminal/help/start_advanced/install_linux'
+const dmt5_macos_url =
+    'https://download.mql5.com/cdn/web/metaquotes.software.corp/mt5/MetaTrader5.dmg'
+const dmt5_windows_url = 'https://download.mql5.com/cdn/web/deriv.limited/mt5/deriv5setup.exe'
+const dp2p_google_play_url =
+    'https://play.google.com/store/apps/details?id=com.deriv.dp2p&hl=en&gl=US'
+const gtm_test_domain = 'deriv-com.binary.sx'
+const livechat_client_id = '66aa088aad5a414484c1fd1fa8a5ace7'
+const livechat_license_id = 12049137
+const map_api_key = 'AIzaSyAEha6-HeZuI95L9JWmX3m6o-AxQr_oFqU'
+const mga_link_url =
+    'https://authorisation.mga.org.mt/verification.aspx?lang=EN&company=a5fd1edc-d072-4c26-b0cd-ab3fa0f0cc40&details=1'
+const p2p_playstore_url = 'https://play.google.com/store/apps/details?id=com.deriv.dp2p'
+const pushwoosh_app_code = 'DD293-35A19'
+const sample_rate = 25
+const smarttrader_url = 'https://smarttrader.deriv.com'
+const zoho_url = 'https://deriv.zohorecruit.com'
```

Comment:
Hi @yashim-deriv, Can't we just use 'export' for each line? 

i.e: `export const zoho_url = 'https://deriv.zohorecruit.com' `

Reply:
Yeah we can. Would reduce the number of lines as well 👍🏼